### DISCUSSION TITLE
Check if a link is an rss feed entry you have in your database

### DISCUSSION THREAD

**Discussion by nirodhvana** [jocmp/capyreader#544] [Feature Requests]
This can be useful when reading a blogpost that has a link to a previous blog post and having the link open that previous blogpost instead of opening in a browser.

it would be great if we could open the link in the app even if it is from another feed that has been added to the app.

**Comment by jocmp** [comment]
@nirodhvana I like the idea. I've seen this in other readers before. I'll take a look at how they do it.

**Reply by nirodhvana** [reply]
Thank you ;)
